### PR TITLE
Fixes a bunch more cult bugs

### LINKED ISSUE #27021: A cultist being turned to a cult master creates a visible "HAS REMEMBERED THEIR OLD FAITH" message
@Robustin 

>Braxton Cherry looks like he just reverted to their old faith!

### LINKED ISSUE #27037: Final Reckoning summons shades out of their soulstones
yyyyyup

### LINKED ISSUE #27038: A cultist created while voting on a cult master still has the cult master verb, and can thus try to become a cult master
And succeed.

### LINKED ISSUE #27040: Inaccurate end of round message about cult
> The staff managed to stop the cult! Dark words and heresy are no match for Nanotrasen's finest!
> The cultists' objectives were:
> Objective #1: Sacrifice Morty, the Warden. Success!
> Objective #2: Summon Nar-Sie. Success!

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: The verb to Assert Leadership over the cult has been replaced with an action button that does the same thing, but is much more visible.
/:cl:

Fixes #27021 
Fixes #27040 
Fixes #27038
Closes #27037; it still summons shades, but it doesn't leave them in a GODMODE state.
Maybe fixes another different thing hopefully ha ha

### CODE REVIEW

**Comment by Robustin** [comment]
I messed up the victory condition stuff but most of these issues were from getting cucked by not one but two antag datum refactors after I finished my PR, then Joan's fix, while deeply appreciated, did introduce a couple of these issues as well. 

@ChangelingRain 

I opened #27051 but hadn't seen your PR. We have a little bit of overlap but you could probably just add my changes in 60 seconds so I can close mine. 

I will say that I intentionally made the Leadership button into a verb. I didn't want every newbie cultist going "OOH WHATS THIS DO?" early in the round. However, an action button has its advantages and will probably work out better in the long run so meh.

**Comment by ChangelingRain** [comment]
Yeah some of these were my issues
And yeah, the leadership thing being a button does mean more people will randomly press it, but it being a verb made it really, uh, uncommon.

**Comment by Robustin** [comment]
I mean it was Day 1. People who want to be leader pick that stuff up fast, everyone else... not so much.

File: code/game/gamemodes/cult/cult_comms.dm [hunk 1]
```diff
@@ -84,22 +86,82 @@
...
-	background_icon_state = "bg_demon"
-	buttontooltipstyle = "cult"
-	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_CONSCIOUS
+/datum/action/innate/cult/mastervote
+	name = "Assert Leadership"
+	button_icon_state = "cultvote"
+
+/datum/action/innate/cult/mastervote/IsAvailable()
+	if(GLOB.cult_vote_called)
+		return 0
```
Comment (lzimann) [inline code/game/gamemodes/cult/cult_comms.dm]: false

**Pull request merged**